packages/tools/devtools/devtools-core/src/messaging/devtools-messages/CloseDevtools.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDevtoolsMessage } from "../Messages";
+
+/**
+ * Encapsulates types and logic related to {@link CloseDevtools.Message}.
+ *
+ * @internal
+ */
+export namespace CloseDevtools {
```

Comment:
What do you think about calling this "DevtoolsClosed" (or something like that)? To me "CloseDevtools" sounds like an instruction to close something, rather than a notice that something has already closed.

Reply:
I see @alexvy86 left a similar comment. I like "DevtoolsDisposed" too. Either sounds good to me :)

Reply:
Then I think `DevtoolsDisposed` sounds more accrue. Updated it.